src/DoctrineAuditBundle/Transaction/AuditTrait.php

```diff
@@ -164,7 +165,12 @@ private function summarize(EntityManagerInterface $em, $entity = null, $id = nul
             return null;
         }
 
-        $em->getUnitOfWork()->initializeObject($entity); // ensure that proxies are initialized
+        // ensure that proxies are initialized
+        if (($entity instanceof Proxy) && !$entity->__isInitialized()) {
```

Comment:
I think you should also check if the entity is a `PersistentCollection` and not `isInitialized`.
And btw, you can also switch `$em->getUnitOfWork->initializeObject` by `$em->initializeObject`

Reply:
Good point, i update it